code/modules/shuttle/on_move.dm

```diff
@@ -179,7 +179,7 @@ All ShuttleMove procs go here
 	for(var/obj/machinery/door/airlock/A in range(1, src))  // includes src
 		A.shuttledocked = FALSE
 		A.air_tight = TRUE
-		INVOKE_ASYNC(A, /obj/machinery/door/.proc/close)
+		addtimer(CALLBACK(A, /obj/machinery/door/.proc/close), 0)
```

Comment:
just use a spawn. 0 duration timers are pointless

Reply:
spawn() would do the exact same thing that invoke_async does here. if spawn() runtimes, the entire proc runtimes and cancels out. if invoke_async runtimes, the entire proc runtimes and cancels out. meanwhile, if addtimer() runtimes, the only thing that gets cancelled out is the proc called by the timer, while the rest of the proc that created the timer can continue running without issue.
spawn() and invoke_async handle everything within the proc itself. addtimer outsources the function to the timer subsystem proper.